### ISSUE TITLE
Why not change the training mode to 'False' in the line # 339?

### ISSUE THREAD

**Issue by zem007** [HasnainRaz/FC-DenseNet-TensorFlow#9]
Hi, thanks for your sharing. I learned a lot!
I am little confused for the training mode in the Model.py (line#: 339). To get the dice for the validation set, we should change the training mode from 'True' to 'False', is that right? Because we we have trained the weights by the training set, and we should not train any weights by the validation set.

I am a new learner. I will be very grateful if you reply me. 
Thanks a lot!

**Comment by HasnainRaz** [comment]
Thanks for the interest, the variable "training" only denotes under what mode should the dropout and the batch_norm layers work, it doesn't enable or disable training. If it is set to true, batch_norm is parameters are calculated for each batch, if it is set to false, the batch_norm parameters learned during training are used, the author's in the originally paper say that they set this to True even when evaluating.

You can of course set it to False, and you are correct that it should be False, but I set it True because in the paper the authors set it True just because it gives better results. Which might not be the True in your case.
